genegram/predict.py

```diff
@@ -152,6 +147,26 @@ def get_stem_len(i0, j0, img):
     return img
 
 
+# set each gray pixel of network prediction to black/white
+# according to threshold coeff
+def img_binarize(img, coeff=0.6):
```

Comment:
In the future, we should make the binarization coefficient configurable by the user. Maybe just add yet another CLI parameter.